src/include/network/network_util.h

```diff
@@ -51,6 +51,15 @@ class NetworkUtil {
 
   /**
    * @param type query type from the parser
+<<<<<<< HEAD
```

Comment:
Weird artifact from a merge? I think the function is dead code anyway right now.